### PR TITLE
chore: update dependencies

### PR DESCRIPTION
**Pull request by hansieodendaal**
- Updated "tari_bulletproofs_plus".
- Updated "tari_utilities".

### CODE REVIEW

**Comment by AaronFeickert** [comment]
There are several (non-breaking) Bulletproofs+ fixes that aren't included in this PR's dependency revision. Might be worth waiting until a new version is tagged and released; then it can be referenced by crate version.

**Comment by hansieodendaal** [comment]
> There are several (non-breaking) Bulletproofs+ fixes that aren't included in this PR's dependency revision. Might be worth waiting until a new version is tagged and released; then it can be referenced by crate version.

Absolutely, this can be referenced by commit until all is stable.

**Comment by AaronFeickert** [comment]
There's still [one pending PR](https://github.com/tari-project/bulletproofs-plus/pull/50) that hasn't been merged, but it might be worthwhile to update to the [latest commit](https://github.com/tari-project/bulletproofs-plus/commit/b253a6fa4166e821a2103b2c4415bad2dfabf86d) for CI purposes.

**Comment by AaronFeickert** [comment]
Crazy timing, but the pending PR was just merged, so there's a [newer commit](https://github.com/AaronFeickert/bulletproofs-plus/commit/b4756b05edfc2cc0f9b0240ca34a22bd9730fbd1) now.

**Comment by AaronFeickert** [comment]
I assume a new BP+ version will be issued after any audit findings are addressed.

File: Cargo.toml [hunk 1]
```diff
@@ -11,10 +11,10 @@ version = "0.17.0"
 edition = "2018"
 
 [dependencies]
-tari_utilities = { version = "0.5", features = ["zero", "std"] }
+tari_utilities = { package = "tari_utilities", version = "0.5.1", features = ["zero", "std"] }
```
Comment (AaronFeickert) [inline Cargo.toml:14]: Do you need the `package` specification here?

**Review by AaronFeickert — CHANGES_REQUESTED** [review]

File: Cargo.toml (hunk 1 above)
Comment (hansieodendaal) [reply Cargo.toml:14]: Not necessary, will remove

**Review by SWvheerden — APPROVED** [review]

**Pull request merged**